### ISSUE TITLE
Dilute Sulfuric Acid?

### ISSUE THREAD

**Issue by bHXDgiuMa5yLHyC8** [Blood-Asp/GT5-Unofficial#1440]
In general, I like realistic chemistry, but I don't understand what purpose this fluid serves. Can't we just abstract it away by having the recipes that produce it consume or produce half as much sulfuric acid?

**Comment by Antillar** [comment]
In chemical reaction water is created and sulfuric acid is created. They are created in one reaction and therefore cannot be separated, mixing into diluted sulfuric acid which is 50% sulfuric acid 50% water.

**Comment by nictrace** [comment]
yes, acids are consumed in reaction, but not 100%. We can recuperate 50% of acids back...
